cypress/fixtures/tequila_list.json

```diff
@@ -0,0 +1,64 @@
+{
+  "ProductSearchResults": [
```

Comment:
No sure about this key. Why not just products?

Reply:
Have corrected this to 'results', misunderstood what the backend had said until I ran a binding pry on the code to verify the output.